src/Settings/SettingsHelper.php

```diff
@@ -600,6 +600,12 @@ private function prepareFormViewDetails(array $data): array
 	 */
 	private function sortFields($a, $b): bool
```

Comment:
This has to be the worst naming of variables in the history of variable naming 🙈